guides/doc-Upgrading_and_Updating/topics/con_upgrade-paths.adoc

```diff
@@ -0,0 +1,38 @@
+[[upgrade_paths]]
+== Upgrade Paths
+
+ifdef::satellite[]
+You can upgrade to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVersionPrevious}.
+
+{ProjectServer}s and {SmartProxyServers} on earlier versions must first be upgraded to {Project} {ProjectVersionPrevious}.
+For more information, see the https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html/upgrading_and_updating_red_hat_satellite/[_{UpgradingDocTitle} to {ProjectVersionPrevious}_].
+endif::[]
+
+ifndef::satellite[]
+You can upgrade to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVersionPrevious}.
+
+{ProjectServer}s and {SmartProxyServers} on earlier versions must first be upgraded to {Project} {ProjectVersionPrevious}.
+For more information, see the {Project} {ProjectVersionPrevious} Upgrade documentation.
+endif::[]
```

Comment:
I'd prefer to have some attribute like `UpgradingPreviousDocURL` (and `UpgradingPreviousDocTitle`) to the URL. We can easily determine that and then unify both these texts.